composer.json

```diff
@@ -14,7 +14,6 @@
     "api-clients/hydrator": "^1.0.1",
     "api-clients/middleware": "^4.0 || ^3.0 || ^2.0 || ^1.1",
     "api-clients/transport": "^3.0 || ^2.0 || ^1.0",
-    "php-di/php-di": "^5.4.3",
```

Comment:
Can you revert this, PHP-DI was specifically picked for the autowiring of dependencies for services and handlers used throughout the entire @php-api-clients project. External definitions are supported through https://github.com/php-api-clients/foundation/blob/master/src/Factory.php#L53 but I would like to support wrappoing an external PSR-11 container in the future.